# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/cookiebanners/moz.build
# Commit: 7976b0cb00d6
# Full Hash: 7976b0cb00d60fe63424c3572e0c64a6ec0bebcb
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-03-29 20:59:04
# Regressor Bug: 1874741
# File Overlap Count: 1
# Description:
#   Bug 1874741 - Part 3: Implement the telemetry to collect GDPR choice on Google Search. r=anti-tracking-reviewers,pbz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205443
# ==============================================================================

diff -r 1aa93711fe2f -r 7976b0cb00d6 toolkit/components/cookiebanners/moz.build
--- a/toolkit/components/cookiebanners/moz.build	Fri Mar 29 13:03:05 2024 +0000
+++ b/toolkit/components/cookiebanners/moz.build	Fri Mar 29 13:03:06 2024 +0000
@@ -14,6 +14,7 @@
     "nsICookieBannerListService.idl",
     "nsICookieBannerRule.idl",
     "nsICookieBannerService.idl",
+    "nsICookieBannerTelemetryService.idl",
     "nsICookieRule.idl",
 ]
 
@@ -34,6 +35,7 @@
     "nsClickRule.h",
     "nsCookieBannerRule.h",
     "nsCookieBannerService.h",
+    "nsCookieBannerTelemetryService.h",
     "nsCookieInjector.h",
     "nsCookieRule.h",
 ]
@@ -44,6 +46,7 @@
     "nsClickRule.cpp",
     "nsCookieBannerRule.cpp",
     "nsCookieBannerService.cpp",
+    "nsCookieBannerTelemetryService.cpp",
     "nsCookieInjector.cpp",
     "nsCookieRule.cpp",
 ]